src/lib/config.js

```diff
@@ -3,6 +3,7 @@ const RC = require('rc')('CLEDG', require('../../config/default.json'))
 module.exports = {
   HOSTNAME: RC.HOSTNAME.replace(/\/$/, ''),
   PORT: RC.PORT,
+  MAX_FULFIL_TIMEOUT_DURATION_SECONDS: RC.MAX_FULFIL_TIMEOUT_DURATION_SECONDS,
```

Comment:
I.e. I feel that it would be better to have the default value set here.

Reply:
Yes. It's better. done.